cal-itp/graas
# persist vehicle position feed, cache alert feed, create comprehensive feed tests

Changes:
- persist vehicle position feed. vehicle position feed state is now held in DB instead of local server instances
- cache alert feed for one minute instead of always reloading from DB
- comprehensive vehicle position feed test that includes posting updates, reading feed and complaining if feed doesn't match updates
- comprehensive service alert feed test that includes posting updates, reading feed and complaining if feed doesn't match updates

Tested against local server:
- bus app: scanning qr code, selecting trip and posting updates
- dispatch UI: loading assignments using both `Today` and `Tomorrow` button, drag vehicles onto blocks, clear vehicle assignment from blocks, deploy assignments, verify that assignments persist after deployment, show block descriptions by clicking on blocks
- run vehicle position test
- run service alert test
